octokit-project/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,22 @@
+package org.example;
+
+public class Main {
+
+    public static void main(String[] args) {
+        int a = 1;
+        float b = 2.0f;
+        double c = 3.0;
+        long d = 4L;
+        short e = 5;
+        byte f = 6;
+        String g = "Hello";
+
+        System.out.println("a: " + a);
+        System.out.println("b: " + b);
+        System.out.println("c: " + c);
+        System.out.println("d: " + d);
+        System.out.println("e: " + e);
+        System.out.println("f: " + f);
+        System.out.println("g: " + g);
+    }
+}
```

Comment:
[Severity: Low] The main method is currently printing values directly to the console.
Consider implementing a logging framework instead of using System.out.println for better control over logging levels and outputs.
Rationale: This allows for more flexible logging management and is a common practice in production code.
Reference: https://www.baeldung.com/java-logging